TEAMS.md

```diff
@@ -0,0 +1,35 @@
+# Samvera GitHub Teams
+
+Between the [Samvera GitHub Organizations](./ORGS.md), users are structured into the following teams:
+
+## Samvera
+
+| Team | Description |
+| ---- | ----------- |
+| [admins](https://github.com/orgs/samvera/teams/admins) | Users with administrative privileges for the Samvera Organization |
+| [contributors](https://github.com/orgs/samvera/teams/contributors) | Users with the privileges to manage releases for GitHub projects, as well as publish releases to RubyGems |
+| [emerita](https://github.com/orgs/samvera/teams/emerita) | Users who are no longer active within the Samvera Community, but who have contributed to community projects |
+| [maintenance](https://github.com/orgs/samvera/teams/maintenance) | Current, active members of the Component Maintenance Interest Group |
+
+## Samvera Labs
+| Team | Description |
+| ---- | ----------- |
+| [admins](https://github.com/orgs/samvera-labs/teams/admins) | Users with administrative privileges for the Samvera Labs Organization |
+| [contributors](https://github.com/orgs/samvera-labs/teams/contributors) | Users with the privileges to manage releases for GitHub projects, as well as publish releases to RubyGems |
+| [maintenance](https://github.com/orgs/samvera-labs/teams/maintenance) | Current, active members of the Component Maintenance Interest Group |
+
+## Samvera Deprecated
+| Team | Description |
+| ---- | ----------- |
+| [admins](https://github.com/orgs/samvera-deprecated/teams/admins) | Users with administrative privileges for the Samvera Deprecated Organization |
+| [contributors](https://github.com/orgs/samvera-deprecated/teams/contributors) | Users with the privileges to manage releases for GitHub projects, as well as publish releases to RubyGems |
+| [maintenance](https://github.com/orgs/samvera-deprecated/teams/maintenance) | Current, active members of the Component Maintenance Interest Group |
+
+## Administrators (Admins)
+GitHub Organizations distinguish between [owners](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/maintaining-ownership-continuity-for-your-organization) and [moderators](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/managing-moderators-in-your-organization). Members of the `admins` Team are possess both of these privileges.
+
+## Contributors
```

Comment:
Can you add something to this section about how this team is sync'd to rubygems so that members have push privileges as stated in the chart above?